troubleshooting/troubleshooting-ios-cannot-load-iframe.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Cannot Load iframe on iOS devices when using Cordova 5.0.0 or later
+description: Cannot load iframe on iOS devices with Cordova 5.0.0 and 5.2.0.
+type: troubleshooting
+page_title: Cordova 5.0.0 or later apps do not load iframe on iOS
+slug: iframe-not-loading-ios
+position: 
+tags: appbuilder, ios, cordova 
+teampulseid: 
+ticketid: 1060248
+pitsid: 
+---
+
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Code (AppBuilder)</td>	
+  </tr>
+  <tr>
+    <td>Mobile development type</td>
+    <td>Hybrid (Cordova app)</td>	
+  </tr>
+
+  <tr>
+    <td>Mobile OS</td>
+    <td>iOS</td>	
+  </tr>
+  <tr>
+    <td>Cordova framework version</td>
+    <td>Cordova 5.0.0 and above</td>	
+  </tr>
+</table>
+
+## Description
+
+Apps that are deployed on an iOS device don't show iframe's content.
+
+## Steps to Reproduce
+
+1. Create a Cordova application.
+1. Set the **Cordova version** to be equal or greater than **5.0.0** from **Properties->Cordova** Version.
+1. Include a `<iframe src=""></iframe>` tag to a HTML view.
```

Comment:
In both places you should use "an" instead of "a". The easiest way to not mix those two up is to try and say it out loud. Does "a aifreim" or "an aifreim" sound better? Does "a eichtiemel" or "an eichtiemel" sound better?